.rubocop.yml

```diff
@@ -209,3 +209,8 @@ Lint/ConstantOverwrittenInRescue: # new in 1.31
   Enabled: true
 Lint/NonAtomicFileOperation: # new in 1.31
   Enabled: true
+
+RSpec/Capybara/SpecificMatcher: # new in 2.12
+  Enabled: true
+RSpec/Rails/HaveHttpStatus: # new in 2.12
+  Enabled: true
```

Comment:
no actual code changes resulted from this, so i figured it was ok to throw on one small extra commit to get rid of the warnings, but happy to split out if preferred.

Reply:
I think it's kind of odd to enable Rails cops for this repo, which doesn't have a dependency on Rails, but it doesn't really matter much.

Reply:
good point, dropped